test/bigtest/interactors/view-request.js

```diff
@@ -35,6 +35,10 @@ import { contains } from './helpers';
   staffNotesAccordion = new NotesAccordion('#staff-notes');
 
   itemAccordionClick = clickable('#accordion-toggle-button-item-info');
+
+  whenNotesLoaded() {
+    return this.when(() => this.staffNotesAccordion && this.staffNotesAccordion.notes().length > 0);
```

Comment:
I would suggest using `isEmpty` from `lodash` instead.
```suggestion
    return this.when(() => this.staffNotesAccordion && !isEmpty(this.staffNotesAccordion.notes()));
```